File: JAVA PROGRAMMING/Test32.java
// 32. PRINT THOSE NUMBER FROM 0 TO N WHICH ARE NOT DIVISIBLE BY THE GIVEN TARGET
import java.util.Scanner;
public class Test32 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range Value: ");
        int range = in.nextInt();
        System.out.println("Enter Target Value: ");
        int target = in.nextInt();
        for(int i = 0; i <= range; i++) {
            if(i % target != 0) {
                System.out.print(i+" ");
            }
        }
        in.close();
    }
}

File: PYTHON PROGRAMMING/Test32.py
# 32 PRINT THOSE NUMBER FROM 0 TO N WHICH ARE NOT DIVISIBLE BY THE GIVEN TARGET
def func(limit, target):
    for i in range(0, limit + 1):
        if(i % target != 0):
            print(i,end=" ")
    print()
func(10, 3)
func(20, 4)
func(30, 7)